src/components/Register.tsx

```diff
@@ -15,58 +15,90 @@ import {
   UnorderedList,
   VStack
 } from "@chakra-ui/react";
-import { ViewIcon, ViewOffIcon } from '@chakra-ui/icons'
+import { ViewIcon, ViewOffIcon } from "@chakra-ui/icons"
 import { useNavigate } from "react-router-dom";
+import { register } from "../utils/fetchData";
+
 
 const Register = () => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
   const [showRequirements, setShowRequirements] = useState(false)
   const [password, setPassword] = useState("");
   const [confirmPassword, setConfirmPassword] = useState("");
-  const [type, setType] = useState('password');
-  const [typeConfirm, setTypeConfirm] = useState('password');
+  const [type, setType] = useState("password");
+  const [typeConfirm, setTypeConfirm] = useState("password");
   const [showPassword, setShowPassword] = useState(false);
 
   const navigate = useNavigate();
-  const navigateToHome = () => {
-    navigate("/home");
-  }
 
   const navigateToLogin = () => {
     navigate("/login");
   }
 
   const handleShowPassword = () => {
-    if (type === 'password') {
-      setType('text')
-      setTypeConfirm('text')
+    if (type === "password") {
+      setType("text")
+      setTypeConfirm("text")
       setShowPassword(true)
     } else {
-      setType('password')
-      setTypeConfirm('password')
+      setType("password")
+      setTypeConfirm("password")
       setShowPassword(false)
     }
   }
 
+  const handleRegister = () => {
+    register(name, email, password, confirmPassword) 
+    .then((data) => {
+      if (data.status === 201) {
+        navigate("/home");
```

Comment:
Could you please move state cleanup here?

Reply:
Thank you! Right, it makes sense!